build.gradle

```diff
@@ -29,6 +29,11 @@ plugins {
 }
 
 apply from: "versions.gradle"
+apply plugin: "com.vanniktech.android.junit.jacoco"
+
+junitJacoco {
+    jacocoVersion = '0.8.4'
```

Comment:
this seems to be related - https://github.com/jacoco/jacoco/issues/968 - i'm not sure what happened between my PR and your PR and why that would break it. I'll add this info to the improvement ticket but i don't really want to lose another day of my life debugging this 😅

Reply:
Sure, yes, lets add a tech debt ticket to upgrade jacoco version.